src/garage/sampler/utils.py

```diff
@@ -11,7 +11,8 @@ def rollout(env,
             max_path_length=np.inf,
             animated=False,
             speedup=1,
-            always_return_paths=False):
+            always_return_paths=False,
+            deterministic=False):
```

Comment:
Please document this new parameter (you don't need to document the old parameters).